game/scripts/npc/abilities/bristleback_quill_spray.txt

```diff
@@ -42,12 +43,12 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "quill_base_damage"                               "20.0 40.0 60.0 80.0 120.0 160.0"
+        "quill_base_damage"                               "20.0 40.0 60.0 80.0 120.0 240.0"
       }
       "03"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "quill_stack_damage"                              "30.0 32.0 34.0 36.0 46.0 56.0"
+        "quill_stack_damage"                              "30.0 32.0 34.0 36.0 40.0 52.0"
```

Comment:
This spell only needs straight buffs. You dont get any damage from this skill. Everyone is walking around with 70% Physical resistance minimum. (I know you have minus armour but you are neever killing people with quill spray.

Reply:
I wanted to buff Quil Spray dmg talent in another PR but I can do it now...